pkgs/development/libraries/cosmopolitan/default.nix

```
@@ -0,0 +1,80 @@
+{ lib, gcc9Stdenv, fetchFromGitHub, runCommand, cosmopolitan }:
+
+gcc9Stdenv.mkDerivation {
+  pname = "cosmopolitan-unstable";
+  version = "20210320";
+
+  src = fetchFromGitHub {
+    owner = "jart";
+    repo = "cosmopolitan";
+    rev = "9c81729008e9e29fdfdbfe3b245abaff5c35ec6e";
+    sha256 = "sha256-Wa/GAzRsUn5rkwPq7njKCqfNbufNdU/h1T9JtUotKf8=";
+  };
```

0.3's out, let's have that
```suggestion
gcc9Stdenv.mkDerivation rec {
  pname = "cosmopolitan";
  version = "0.3";

  src = fetchFromGitHub {
    owner = "jart";
    repo = "cosmopolitan";
    rev = version;
    sha256 = "sha256-OVdOObO82W6JN63OWKHaERS7y0uvgxt+WLp6Y0LsmJk=";
  };
```